cmd/nerdctl/image_convert.go

```diff
@@ -206,7 +206,17 @@ func getESGZConvertOpts(cmd *cobra.Command) ([]estargz.Option, error) {
 		estargz.WithCompressionLevel(estargzCompressionLevel),
 		estargz.WithChunkSize(estargzChunkSize),
 	}
+
+	experimental, err := cmd.Flags().GetBool("experimental")
+	if err != nil {
+		return nil, err
+	}
+
 	if estargzRecordIn != "" {
+		if !experimental {
+			return nil, fmt.Errorf("estargz-record-in only work with enable experimental feature")
```

Comment:
```suggestion
			return nil, fmt.Errorf("estargz-record-in requires experimental mode to be enabled")
```

Reply:
(Same applies to all other flags in this PR)

Reply:
new implement of checkExperimental will log the experimental feature name.

Reply:
Still has grammatical errors